molo/core/tests/test_commands.py

```diff
@@ -88,25 +88,33 @@ def test_add_translated_pages_to_pages(self):
         # run command to add languages to these pages
         call_command('add_language_to_pages')
 
+        # remove the langauge from one page to ensure script works
+        # as intended for pages without a language
+        # which is to not add them as a translated page
+
+        french_article.language = None
+        french_article.save()
+
+        self.assertIsNone(french_article.language)
         # run command to add list of translated_pages to these pages
         call_command('add_translated_pages_to_pages')
 
         # test translated pages for articles
         self.assertTrue(english_article.translated_pages.filter(
             pk=spanish_article.pk).exists())
-        self.assertTrue(english_article.translated_pages.filter(
+        self.assertFalse(english_article.translated_pages.filter(
             pk=french_article.pk).exists())
         self.assertFalse(english_article.translated_pages.filter(
             pk=english_article.pk).exists())
         self.assertTrue(spanish_article.translated_pages.filter(
             pk=english_article.pk).exists())
-        self.assertTrue(spanish_article.translated_pages.filter(
+        self.assertFalse(spanish_article.translated_pages.filter(
             pk=french_article.pk).exists())
         self.assertFalse(spanish_article.translated_pages.filter(
             pk=spanish_article.pk).exists())
-        self.assertTrue(french_article.translated_pages.filter(
+        self.assertFalse(french_article.translated_pages.filter(
```

Comment:
are we sure we want the french article not to have any translated pages if it is the one without a language?

Reply:
yes, because if it has no language it is a page that cannot be translated

Reply:
okey dokey